backend/tracim_backend/lib/crud_hook/hookspec.py

```diff
@@ -90,3 +97,35 @@ def on_workspace_subscription_deleted(
         self, subscription: WorkspaceSubscription, context: TracimContext
     ) -> None:
         ...
+
+    @hookspec
+    def on_reaction_created(self, reaction: Reaction, context: TracimContext) -> None:
```

Comment:
I don't understand why code about reaction is added in this PR.

Reply:
the reaction hooks were added without defining their specs, I saw it while adding mine so I added the reaction ones too.